examples/crm/src/providers/fakerest/dataProvider.ts

```diff
@@ -22,7 +22,7 @@ import { getContactAvatar } from '../commons/getContactAvatar';
 import { authProvider, USER_STORAGE_KEY } from './authProvider';
 import { withSupabaseFilterAdapter } from './internal/supabaseAdapter';
 
-const baseDataProvider = simpleRestProvider('http://localhost:4000');
+const baseDataProvider = simpleRestProvider('https://localhost:4000');
```

Comment:
As this is never accessed anyway, why not even put something that looks like a real api?
```suggestion
const baseDataProvider = simpleRestProvider('https://crm.api.marmelab.com');
```

Reply:
I thought about it but wasn't sure that it was a good idea